gulpfile.mjs

```diff
@@ -184,8 +187,15 @@ gulp.task('styles', () => {
     return gulp.src(src, { base })
     .pipe(changed(dest))
     .pipe(postcss([
-        postcssPresetEnv()
+        postcssPresetEnv(),
+        cssnano()
     ]))
+    .pipe(rev())
+    .pipe(gulp.dest(dest))
+    .pipe(rename({
```

Comment:
Is this rename just for the manifest file?

Reply:
I suppose I have two questions. First, does `rev` need to happen after `postcss`? In the README instructions for `gulp-rev` they show it as running directly after the src is declared. Second is line 194 actually needed? You typically only want to have one destination at the very end of your stream. It almost looks like that is why you are needing to rename the file.

Reply:
Ok so rereading the docs on `gulp-rev` looks like they do not support streaming which means you do have to move the files then create the manifest. That said we can pass in the base path of `dest` as an option to the rev.manifest function then we wouldn't need the gulp-rename dependency.

If you would like I can push a commit with these changes.

Reply:
Thanks for looking into this.

I had some real issues getting the content of the manifest to be `styles/whatever.css` instead of just `whatever.css`, and this rename was the only way I could get it to work.

Absolutely feel free to improve it if you can though.

Thank you!